### PR TITLE
create comment

### PR DESCRIPTION
**Pull request by anhphamdt98**
## Related Tickets:
- [27316](https://edu-redmine.sun-asterisk.vn/issues/27316)

## WHAT this PR does?
- Create comment and reply comment

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.
image

### CODE REVIEW

**Comment by anhphamdt98** [comment]
ready

**Comment by anhphamdt98** [comment]
Có 1 file "have no changes"

File: app/Http/Controllers/CommentController.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+use Illuminate\Http\Request;
+use App\Http\Requests\CommentRequest;
+use App\Models\Comment;
+
+class CommentController extends Controller
+{
+    public function store(CommentRequest $request)
+    {
+        $input = $request->all();
+        Comment::create($input);
```
Comment (hoangminhuet) [inline app/Http/Controllers/CommentController.php:14]:
```suggestion
        Comment::create($request->all());
```

File: app/Http/Requests/CommentRequest.php [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'comment'=>'required',
```
Comment (hoangminhuet) [inline app/Http/Requests/CommentRequest.php:27]:
```suggestion
            'comment' => 'required',
```

File: resources/views/post/commentDisplay.blade.php [hunk 3]
```diff
@@ -0,0 +1,33 @@
+@foreach($comments as $comment)
```
Comment (hoangminhuet) [inline resources/views/post/commentDisplay.blade.php:1]: convention

File: resources/views/post/commentDisplay.blade.php [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+            <div class="col-md-1 avatar-user">
+                <img src="{{ asset('bower_components/bower-package/images/users/user-1.jpg') }}" class="profile-photo-sm" />
+            </div>            
+            <div class="col-md-11 layout-avatar">
+                <a href="#" class="profile-link">{{ $user->name }}</a>
+                <p>{{ $comment->comment }}</p>
+            </div>
+        </div>
+        <a href="" id="reply"></a>
+        @if(is_null($comment->parent_id))
```
Comment (hoangminhuet) [inline resources/views/post/commentDisplay.blade.php:13]: convention

File: resources/views/post/commentDisplay.blade.php [hunk 5]
```diff
@@ -0,0 +1,33 @@
+@foreach($comments as $comment)
+    <div class="display-comment @if($comment->parent_id != null) layout-comment @endif">
```
Comment (hoangminhuet) [inline resources/views/post/commentDisplay.blade.php:2]: convention

**Pull request merged**